src/utils/stringToIconConvertor.ts

```diff
@@ -1,13 +1,36 @@
+// @ts-nocheck
 import React from 'react'
+import * as fontAwesomeIcons from 'react-icons/fa'
```

Comment:
by doing this you are importing all the icons in react - icons.

Reply:
As we are dynamically importing the icons and also user can choose any icon from the font awesome. So we can not limit the icons. That's why i am using this way of importing bro

Reply:
https://medium.com/@magbicaleman/intro-to-dynamic-import-in-create-react-app-6305bb397c46
https://reactjs.org/docs/code-splitting.html
https://nextjs.org/docs/advanced-features/dynamic-import
https://www.npmjs.com/package/react-dynamic-import